src/MessageContainer.js

```diff
@@ -135,10 +135,20 @@ export default class MessageContainer extends React.Component {
     );
   }
 
+  endReachedProps(){
+    let condProps = {}
+    if (this.props.onTopReached){ condProps.onEndReached = this.props.onTopReached }
```

Comment:
@kfiroo, @arturictus Ups, sorry :sweat_smile: :sweat_smile:. I didn't realize that messages `ListView` is using a [`InvertibleScrollView`](https://github.com/exponentjs/react-native-invertible-scroll-view). I thought that it was a `ScrollView`